projects/igniteui-angular/src/lib/grids/filtering/base/grid-filtering-row.component.ts

```diff
@@ -278,6 +279,10 @@ export class IgxGridFilteringRowComponent implements AfterViewInit, OnDestroy {
         }
     }
 
+    public get weekStart(): number {
```

Comment:
Is this used somewhere at all?

Reply:
I think it is save to remove it, since I can't find this piece of code in my local branch 🤔

Reply:
OK, please remove it :)